src/components/history-page/entry-menu/entry-menu.tsx

```diff
@@ -40,23 +44,24 @@ export const EntryMenu: React.FC<EntryMenuProps> = ({ id, title, location, isDar
           <Trans i18nKey="landing.history.menu.recentNotes"/>
         </Dropdown.Header>
 
-        <ShowIf condition={ location === HistoryEntryOrigin.LOCAL }>
+        <ShowIf condition={ origin === HistoryEntryOrigin.LOCAL }>
           <Dropdown.Item disabled>
             <ForkAwesomeIcon icon="laptop" fixedWidth={ true } className="mx-2"/>
             <Trans i18nKey="landing.history.menu.entryLocal"/>
           </Dropdown.Item>
         </ShowIf>
-        <ShowIf condition={ location === HistoryEntryOrigin.REMOTE }>
+        <ShowIf condition={ origin === HistoryEntryOrigin.REMOTE }>
           <Dropdown.Item disabled>
             <ForkAwesomeIcon icon="cloud" fixedWidth={ true } className="mx-2"/>
             <Trans i18nKey="landing.history.menu.entryRemote"/>
           </Dropdown.Item>
         </ShowIf>
```

Comment:
It would be nice to extract and deduplicate these components to increase the readability of the component.

Reply:
See my comment below regarding splitting and restructuring components.